vllm/model_executor/model_loader.py

```diff
@@ -38,6 +38,16 @@
     "YiForCausalLM": YiForCausalLM,
 }
 
+# Models to be disabled in ROCm
+_ROCM_DISABLED_MODELS = [
+    # ROCm's flash attention does not support sliding window attention
+    # in models such as Mistral
+    "MistralForCausalLM",
+]
```

Comment:
Do you happen to know when the sliding window will be supported by AMD? If it's not gonna happen in the near future, what about supporting the model without the sliding window (and print a warning msg to the users)? While this will effectively limit the maximum context length of Mistral to 4096, I think many users can still use this model. This doesn't need to be fixed in this PR. I'm just wondering your thoughts.

Reply:
The AMD will not support flash-attention v2.3 with sliding windows in near future, as stated in https://github.com/ROCmSoftwarePlatform/flash-attention/issues/22#issuecomment-1831947643 
Sure. we can support the model without the sliding windows (and print a warning msg to the users).